stl/inc/xstring

```diff
@@ -2361,8 +2364,14 @@ public:
     using const_reverse_iterator = _STD reverse_iterator<const_iterator>;
 
 private:
-    static constexpr auto _BUF_SIZE   = _Scary_val::_BUF_SIZE;
-    static constexpr auto _ALLOC_MASK = _Scary_val::_ALLOC_MASK;
+    static constexpr auto _BUF_SIZE              = _Scary_val::_BUF_SIZE;
+    static constexpr auto _ALLOC_MASK            = _Scary_val::_ALLOC_MASK;
+    static constexpr auto _SMALL_STRING_CAPACITY = _Scary_val::_SMALL_STRING_CAPACITY;
+
+    _NODISCARD static _CONSTEXPR20 bool _Entails_large_string(size_type _Size_or_capacity) noexcept {
+        // decides wether planned size/capacity entails large string mode

```

Comment:
>wether

I believe it is a typo and it should be "whether"

Reply:
Yes 😳, and the whole pr might need some rewordings... I'm pretty uncomfortable with my English...

Reply:
fixed in a new push